targets/TARGET_STM/TARGET_STM32F3/TARGET_STM32F303xE/device/TOOLCHAIN_ARM_STD/stm32f303xe.sct

```diff
@@ -56,6 +56,10 @@ LR_IROM1 MBED_APP_START MBED_APP_SIZE  {    ; load region size_region
    .ANY (+RW +ZI)
   }
 
+  RW_IRAM2 (0x10000000) (0x4000)  {  ; RW data
```

Comment:
Could you reorder this? It's confusing because you've put the CCM RAM in between `RW_IRAM1` and `ARM_LIB_STACK`, which are adjacent.

It's probably a good idea to name the execution region `CCMRAM` too, if you're manually directing stuff there. `RW_IRAM2` could be construed as general-purpose RAM.

(I'm assuming you're not viewing it as general-purpose RAM, or you would be directing general stuff there. Comment should probably say that rather than "RW data").

Reply:
Current way follows the naming and location in F437xG.

Reply:
Feel free to fix that too.

Reply:
Ooh, 437's sct is a mess. It's got 5 RAM regions, three of which are adjacent, but they're all separated in the map.

The three together are (`RW_m_crash_data`, `RW_IRAM1`, `ARM_LIB_STACK`) in that order, then `RW_IRAM2` (CCMRAM) and `RW_IRAM3` (BKPSRAM) are separate.

Reply:
fixed the order and renamed to CCMRAM